src/index.html

```diff
@@ -14,37 +14,39 @@
 </head>
 
 <body>
-  <header class="o-header">
+  <header class="c-header">
     <!-- Brand -->
-    <div class="c-brand-logo">
+    <span class="c-brand-logo">
       <img src="assets/Logo@2x.png" alt="bork co.">
-    </div>
+    </span>
 
     <!-- Mobile Navigation -->
-    <div class="o-nav__mobile">
+    <nav class="o-nav__mobile">
       <img class="menu-icon" src="./assets/MenuIcon2x.png" alt="Menu">
-    </div>
+    </nav>
 
     <!-- Desktop Navigation -->
-    <div class="o-nav__desktop">
+    <nav class="o-nav__desktop">
       <!-- Nav Items -->
-      <ul class="o-nav__items">
+      <ul class="o-nav__items clean-list">
         <li><a href="#">Locations</a></li>
         <li><a href="#">Policies</a></li>
         <li><a href="#">Contact</a></li>
         <li><a class="c-button c-button--secondary" href="#">Log In</a></li>
       </ul>
-    </div>
+    </nav>
   </header>
 
-  <div class="o-content">
+  <main>
     <!-- Main Hero -->
-    <div class="o-hero">
-      <h1>Wherever you need to go, we'll keep your pooch safe, and happy.</h1>
-      <a  class="c-button c-button--primary c-booking-button" href="#" rel="noopener noreferrer">Book Now</a>
-      <div class="c-booking">
+    <section class="o-hero">
+      <header class="c-booking__header">
+        <h1>Wherever you need to go, we'll keep your pooch safe, and happy.</h1>
+        <a class="c-button c-button--primary c-booking-button" href="#" rel="noopener noreferrer">Book Now</a>
+      </header>
+      <article class="c-booking">
```

Comment:
I'm curious about the use of `article` here. Why do you decide to go with this tag?

Reply:
My thought on this, was that the booking element is used on the mobile experience as another page, because this section disappears on mobile and is only a Book Now button, which is why I was thinking article... but it could definitely be a section.